lib/node_modules/@stdlib/array/base/linspace2d/README.md

```diff
@@ -0,0 +1,128 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# linspace2d
+
+> Generate a linearly spaced two-dimensional nested numeric array.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var linspace2d = require( '@stdlib/array/base/linspace2d' );
+```
+
+#### linspace2d( start, stop, shape, colexicographic )
+
+Generates a linearly spaced two-dimensional nested numeric array.
+
+```javascript
+var x = linspace2d( 0, 100, [ 2, 3 ], false );
+// returns [ [ 0, 20, 40 ], [ 60, 80, 100 ] ]
+
+x = linspace2d( 0, 100, [ 2, 3 ], true );
+// returns [ [ 0, 40, 80 ], [ 20, 60, 100 ] ]
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   The function assumes that the product of the number of rows and columns is greater than or equal to `2`.
+
+-   The output `array` is guaranteed to include the `start` and `stop` values. Beware, however, that values between `start` and `stop` are subject to floating-point rounding errors. Hence,
+
+    ```javascript
+    var arr = linspace2d( 0, 10, [ 2, 2 ], false );
+    // returns [ [ 0, 3.3333333333333335 ], [ 6.666666666666667, 10 ] ]
+    ```
+
+    If you desire more control over element precision, consider using [`roundn`][@stdlib/math/base/special/roundn]:
+
+    ```javascript
+    var roundn = require( '@stdlib/math/base/special/roundn' );
+
+    // Create an array subject to floating-point rounding errors:
+    var arr = linspace( 0, 10, [ 3, 3 ], false );
+
+    // Round each value to the nearest hundredth:
+    var i;
```

Comment:
Use [`map2d`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/array/base/map2d) here in order to avoid nested loops. This will also allow you to use a `// returns` annotation, rather than `console.log` here.